jwst/ami/matrix_dft.py

```diff
@@ -18,23 +20,22 @@
 you replace "fast calculations on very large arrays" with "relatively slow
 calculations on much smaller ones".
 
-Code originally by A. Sivaramakrishnan
-2010-11-05 Revised normalizations for flux conservation consistent
-with Soummer et al. 2007. Updated documentation.  -- M. Perrin
-2011-2012: Various enhancements, detailed history not kept, sorry.
-2012-05-18: module renamed SFT.py -> matrixDFT.py
-2012-09-26: minor big fixes
-2015-01-21: Eliminate redundant code paths, correct parity flip,
-PEP8 formatting pass (except variable names)-- J. Long
+Notes
+-----
+Code originally by A. Sivaramakrishnan, with early modifications by
```

Comment:
@emolter and/or @melanieclarke , does this look good to you now?

Reply:
I'd simplify this further and just attribute original code authorship to A. Sivaramakrishnan, M. Perrin, and J. Long, and then reference Soummer paper separately.

Right now the wording is unclear - "in accordance to" is an unusual construction, and the fact that J. Long comes after the reference makes it seem like J. Long is another reference, not another code author

Reply:
Are you able to provide desired text as suggestion? Thanks!

Reply:
Nvm I did a best effort interpretation. Hopefully good enough.